config/config.go

```diff
@@ -0,0 +1,84 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/BurntSushi/toml"
+)
+
+const (
+	DefaultDirName = ".dalc"
+	ConfigFileName = "dalc.toml"
+)
+
+var (
+	DefaultConfigPath string
+)
+
+func init() {
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		log.Fatal(err)
+	}
+	DefaultConfigPath = fmt.Sprintf("%s/%s/", homeDir, DefaultDirName)
+}
+
+type ServerConfig struct {
+	BlockSubmitterConfig `toml:"block-submitter"`
+	LightClientConfig    `toml:"light-client"`
+	KeyringConfig        `toml:"keyring"`
+}
+
+func (cfg ServerConfig) Save(path string) error {
+	cfgFile, err := os.OpenFile(path, os.O_CREATE|os.O_RDWR, 0660)
+	if err != nil {
+		return err
+	}
+	err = toml.NewEncoder(cfgFile).Encode(cfg)
+	if err != nil {
+		return err
+	}
+	return cfgFile.Close()
+}
+
+func Load(path string) (ServerConfig, error) {
+	var cfg ServerConfig
+	rawCfg, err := os.ReadFile(path)
+	if err != nil {
+		return cfg, err
+	}
+	_, err = toml.Decode(string(rawCfg), &cfg)
+	if err != nil {
+		return cfg, err
+	}
+	return cfg, nil
+}
```

Comment:
I didn't write tests for these, hope that alright considering they're super basic

Reply:
I created (good-first-)issue for this: https://github.com/celestiaorg/dalc/issues/14